sources/Foundation/Types/Types.h

```diff
@@ -179,6 +179,7 @@ struct FourCC {
     InstrumentName = 148,
     ActionRenderMixdown = 149,
     ActionRenderStems = 150,
+    SampleInstrumentSlices = 151,
```

Comment:
We've got I think now 3 PRs in progress which all add more so depending which get merged first this will need to be updated.